common/src/main/java/com/wynntils/features/user/HorseMountFeature.java

```diff
@@ -33,10 +39,22 @@ public class MountHorseHotkeyFeature extends UserFeature {
     private static boolean alreadySetPrevItem = false;
 
     @RegisterKeyBind
-    private final KeyBind mountHorseKeyBind =
-            new KeyBind("Mount Horse", GLFW.GLFW_KEY_R, true, this::onMountHorseKeyPress);
+    private final KeyBind mountHorseKeyBind = new KeyBind("Mount Horse", GLFW.GLFW_KEY_R, true, this::mountHorse);
 
-    private void onMountHorseKeyPress() {
+    @SubscribeEvent
```

Comment:
Maybe this new functionality should be optional? People might want to use the hotkey without wanting this functionality.

Reply:
@magicus fixed